### PR TITLE
Updated drstrace URL

### PR DESCRIPTION
**Pull request by DebuggingSystems**
Noticed the 404 while searching documentation.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,7 +36,7 @@ Tools built on DynamoRIO and available in the [release package](https://dynamori
   - [Function call tracing](https://dynamorio.org/sec_drcachesim_tools.html#sec_tool_func_view)
 - The legacy processor emulator
   [drcpusim](https://dynamorio.org/page_drcpusim.html)
-- The "strace for Windows" tool [drstrace](http://drmemory.org/strace_for_windows.html)
+- The "strace for Windows" tool [drstrace](https://dynamorio.org/page_drstrace.html)
```
Comment (derekbruening) [inline README.md:39]: I think we'd prefer to point here: https://drmemory.org/page_drstrace.html

Comment (derekbruening) [inline README.md:39]: Plus, the "this link" at https://dynamorio.org/page_drstrace.html which is trying to work for a local file view (a downloaded unpacked zip package), fails online: best to also provide https://drmemory.org/page_drstrace.html there in addition to the local-file link so at least one of them will work.

Comment (DebuggingSystems) [reply README.md]: I'll do so. There are a few others places in the documentation where we have it.

Comment (derekbruening) [reply README.md]: OK, thank you for the contribution.

**Comment by DebuggingSystems** [comment]
Not sure about the ARM64 failing check. I was attempting to run it locally myself on Windows.

File: api/docs/tool.gendox [hunk 2]
```diff
@@ -83,34 +83,44 @@ should point at the local documentation provided with the release package.
...
-href="../../drmemory/drmemory/docs/html/page_drstrace.html">this link</a> should
+\p drstrace is a system call tracing tool for Windows.
+It is part of the
+<a href="http://drmemory.org">Dr. Memory tool suite</a>. It is also
+included with DynamoRIO versions 5.0.0 and higher.
+
+If this documentation is part of a DynamoRIO public release,
+<a href="../../drmemory/drmemory/docs/html/page_drstrace.html">this link</a> should
 point at the local documentation provided with the release package.
+<a href="https://drmemory.org/page_drstrace.html">This one</a> points the online
```
Comment (derekbruening) [inline api/docs/tool.gendox:94]: nit: s/points/points to/

**Review by derekbruening — APPROVED** [review]

**Review by derekbruening — APPROVED** [review]
Thank you again for the pull request!

**Pull request merged**